### PR TITLE
Edit Developer Guide for clarity 

### LINKED ISSUE #330: Ensure all UI images use the same OS
This should be done for consistency.

### LINKED ISSUE #364: Edit Developer Guide for clarity

### PR DESCRIPTION
**Pull request by Hikoya**
Resolves #364 
Resolves #330

### CODE REVIEW

File: docs/diagrams/ListReviewActivityDiagram.puml [hunk 1]
```diff
@@ -5,7 +5,7 @@ start
 'Since the beta syntax does not support placing the condition outside the
 'diamond we place it as the true branch instead.
 if () then ([Command is Valid])
-    :FoodWhere lists all [Review]s;
+    :FoodWhere lists all [Reviews];
```
Comment (clarence-chew) [inline docs/diagrams/ListReviewActivityDiagram.puml:8]: Maybe we should discuss what format to use for this, but initially I would have thought [Review] is the name of the class.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -100,10 +100,10 @@ How the `Logic` component works:
 1. The command can communicate with the `Model` when it is executed (e.g. to add a review).
 1. The result of the command execution is encapsulated as a `CommandResult` object which is returned back from `Logic`.
 
-The Sequence Diagram below illustrates the interactions within the `Logic` component for the `execute("delete 1")` API call.
+The Sequence Diagram below illustrates the interactions within the `Logic` component for the `execute("rdel 1")` API call.
 ![Interactions Inside the Logic Component for the `delete 1` Command](images/DeleteSequenceDiagram.png)
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:104]: Perhaps update this line to `rdel 1` too?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -169,7 +169,7 @@ A `Review`,
 A `Review` contains the following attributes,
 1. a `Name`, which represent the name of the Stall associated with the Review
 1. an `Address`, which represent the address of the Stall associated with the Review
-2. a `Date`, which represent the day, month and year as specified in `DD/MM/YYYY` format
+2. a `Date`, which represent the day, month and year as specified in `DD/MM/YYYY` , or any of the specified formats in [here](https://ay2223s1-cs2103-w14-2.github.io/tp/DeveloperGuide.html#file-format-for-foodwhere)
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:172]: I would suggest to remove the extra space before the comma, and use a relative link `[here](#file-format-for-foodwhere)`.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -226,23 +226,28 @@ This feature is used to find stalls and reviews in FoodWhere by name and/or by t
 * `sfind`: Finds stalls
 * `rfind`: Finds reviews
 
-`sfind` allows users to find stalls in `AddressBook` by names, through matching of input keyword(s) with stall names. Additionally, users can find stalls by tags, through matching of input keyword(s) with stall tags.
+`sfind` allows users to find stalls in `AddressBook` by `name`, through matching of input keyword(s) with stall names. Additionally, users can find stalls by `tags`, through matching of input keyword(s) with stall tags.
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:229]: I'm not sure why tags and names are now in backticks, are these properties on the Stall class?

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -325,7 +330,7 @@ The Edit Review mechanism is facilitated by `REditCommandParser` and `REditComma
 For the command, the feature extends `command`, and is implemented as such:
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:330]: Extends `Command`?

**Review by clarence-chew — COMMENTED** [review]
Here are my concerns.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md:104]: Thanks, missed that out.

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md:229]: Meant to be ** tags ** instead, thanks for the catch

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md:330]: Edited, thanks.

File: docs/diagrams/ListReviewActivityDiagram.puml (hunk 1 above)
Comment (Hikoya) [reply docs/diagrams/ListReviewActivityDiagram.puml:8]: Hmm.. in this case I will just revert to the original formatting then.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md]: Thanks, changed.

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -291,10 +296,10 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 The Delete Review mechanism is facilitated by `AddressBook`. This feature allows the user to delete a review.
 
-For the command, the feature extends `command`, and is implemented as such:
+For the command, the feature extends `Command`, and is implemented as such:
 * `rdel REVIEW_INDEX`
 
-#### Implementation Flow of Review Deleting feature
+#### Implementation flow of Review Deleting feature
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:302]: I would think deleting/editing should be lowercase since these are not class names.

**Review by clarence-chew — COMMENTED** [review]
One more thing I noted:

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md]: Added in latest commit

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -397,12 +399,11 @@ Step 6. `model.sortReviews()` will interact with the model to sort reviews using
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:399]: Lowercase "sorting"

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -287,14 +292,14 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 ### Review Deleting feature
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:293]: "Review deleting feature"?

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -185,30 +185,30 @@ A `Review` contains the following attributes,
...
 
-Step 1. The user launches the application for the first time. FoodWhere will be initialized with the preloaded data.
+Step 1. The user launches the application for the first time. FoodWhere will be initialized with the preloaded data. Ensure that there is an existing `Stall` with index 1.
 
 Step 2. The user executes `radd s/1 d/20-09-2022 c/The food was good, the chicken rice was fresh. r/4` command to create a new `Review` for `Stall` with index 1.
 
 ![AddReview](images/AddReview.png)
 
-#### UML Diagram for Adding Review
+#### UML Diagram for adding Review
```
Comment (clarence-chew) [inline docs/DeveloperGuide.md:211]: I would consider "UML diagram for adding Review" but that means that "diagram" would need to be consistently in lowercase.

**Review by clarence-chew — APPROVED** [review]
This should be quite fine after the last few capitalization fixes.

File: docs/DeveloperGuide.md (hunk 7 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md:399]: Fixed in latest commit

File: docs/DeveloperGuide.md (hunk 8 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md:293]: Fixed in latest commit

File: docs/DeveloperGuide.md (hunk 9 above)
Comment (Hikoya) [reply docs/DeveloperGuide.md:211]: Fixed in latest commit

**Review by clarence-chew — APPROVED** [review]
LGTM!

**Pull request merged**